### PR TITLE
Change #device to #name on MobilityDevice

### PR DESCRIPTION
**Pull request by dfaulken**
Pending #53, Travis will tell us whether I did this correctly.

### CODE REVIEW

File: app/controllers/passengers_controller.rb [hunk 1]
```diff
@@ -56,7 +56,7 @@ def destroy
   def passenger_params
     permitted_params = params.require(:passenger)
                              .permit :name, :address, :email, :phone,
-                                     :wheelchair, :mobility_device, :active,
+                                     :wheelchair, :mobility_device_id, :active,
```
Comment (dfaulken) [inline app/controllers/passengers_controller.rb]: Unrelated to this PR, but was definitely a bug. Wouldn't have been possible before to actually update a passenger's mobility device.

**Comment by dfaulken** [comment]
Closes #50.

File: db/migrate/20170829182335_change_device_to_name_in_mobility_devices.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ChangeDeviceToNameInMobilityDevices < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :mobility_devices, :device, :name
```
Comment (mboneil10) [inline db/migrate/20170829182335_change_device_to_name_in_mobility_devices.rb]: This is a fantastic change.

File: lib/tasks/import/mobility_devices.rake [hunk 3]
```diff
@@ -6,7 +6,7 @@ desc 'Import mobility devices from Access CSV at data/mobilitydevice.csv'
 namespace :import do
   task mobility_devices: :environment do
     CSV.foreach('data/mobilitydevice.csv', headers: true).with_index 2 do |row, line|
-      device = MobilityDevice.new device: row.fetch('name')
+      device = MobilityDevice.new name: row.fetch('name')
```
Comment (mboneil10) [inline lib/tasks/import/mobility_devices.rake]: Why didn't you have to change this from `row.fetch('device')` to what it is now?

Comment (mboneil10) [reply lib/tasks/import/mobility_devices.rake]: Ohhhh. Never mind.

**Pull request merged**